types/nightwatch/test/testActions.ts

```diff
@@ -46,7 +50,15 @@ it('actions.dragAndDrop', () => {
     browser.perform(async () => {
         const sampleElement = element('.element-class');
         const webElement = await sampleElement.getWebElement();
-        return browser.actions().dragAndDrop(webElement, {x: 12, y: 234});
+        browser.actions().dragAndDrop(webElement, {x: 12, y: 234});
+
+        const webElement2 = sampleElement.findElement();
+        isNightwatchAPI(webElement2);
+        browser.actions().dragAndDrop(await webElement2, {x: 12, y: 234});
```

Comment:
Should we also add an example where `webElement2` is passed directly in the actions API?